.github/workflows/commit-lint.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       - uses: actions/checkout@v4.1.1
         with:
           fetch-depth: 0
-      - uses: actions/setup-python@v4
+      - uses: actions/setup-python@v5
```

Comment:
TODO: We might want to pin this to a specific release/tag.

Reply:
Considering the other included actions, I decided to go the other way around: https://github.com/science-computing/butido/pull/331